test/HostActivationTests/GivenThatICareAboutMultilevelSharedFxLookup.cs

```diff
@@ -189,23 +186,24 @@ public void SharedFxLookup_Must_Roll_Forward_Before_Looking_Into_Another_Folder(
                 .HaveStdErrContaining(Path.Combine(_userSelectedMessage, "9999.0.0-dummy2"));
 
             // Add a prerelease dummy version in CWD
```

Comment:
i am still keeping a handful around, to check that we will never pick up these assets from cwd, removing the rest